src/request/components/request-detail-panel-request.tsx

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+/*tslint:disable:no-var-requires */
+const messageProcessor = require('../util/request-message-processor');
+/*tslint:enable:no-var-requires */
+
+import _ = require('lodash');
+import React = require('react');
+
+const getPayloads = (function() {
+    const getItem = messageProcessor.getTypePayloadItem;
+
+    const options = {
+        'web-response': getItem,
+        'web-request': getItem
+    };
+
+    return function(request) {
+        return messageProcessor.getTypeStucture(request, options);
+    };
+})();
+
+class RequestUrl extends React.Component<{url: string}, {}> {
+    public render() {
+        return (
+            <div>
+                <div className='tab-section tab-section-execution-url'>
+                    <div className='flex flex-row flex-inherit tab-section-header'>
+                        <div className='tab-title col-10'>Url</div>
+                    </div>
+                    <div>{this.props.url}</div>
+                </div>
+            </div>
+        );
+    }
+}
+
+class RequestHeaders extends React.Component<{headers: { [key: string]: string }, title: string}, {}> {
```

Comment:
Yes, I should go back and define proper interfaces for the property bags passed to the component.  It would then look something like:

```javascript
export interface IRequestHeadersProperties {
    headers: { [key: string]: string };
    title: string;
}

class RequestHeaders extends React.Component<IRequestHeadersProps, {}> { }
```

I just did it the fastest way to remove the TypeScript compiler/linter errors.